_data/current.yml

```diff
@@ -1,3 +1,10 @@
+- name: Odin Testkonferansen Norway 2024
+  location: Oslo
+  dates: "September 26, 2024"
+  url: www.https://event.dataforeningen.no/odin/
+  twitter: N/A
```

Comment:
If there is no twitter, just remove the line

Reply:
Ok, there is no twitter

Reply:
Remove line number 5. No reason to have it in.